R/bsync_utils.R

```diff
@@ -301,9 +295,13 @@ bs_gen_dm_nmecr <- function(nmecr_baseline_model, x,
     "Hourly" = "Hour",
     "SLR" = "2 parameter simple linear regression",
     "Three Parameter Heating" = "3 parameter heating change point model",
```

Comment:
@macintoshpie should we remove the redundant ones?  i.e. I assume the keys with 'Three Parameter Heating' and '3PH' are the same, and you are probably only using '3PH'...unless I missed something.

Reply:
We need both from what I can tell b/c it depends on what you pass to nmecr (which allows both)
```r
model <- nmecr::model_with_CP(bsyncr_df,
                                nmecr::assign_model_inputs(regression_type = "Three Parameter Cooling"))
print(model$model_input_options$regression_type)
# prints "Three Parameter Cooling"
model <- nmecr::model_with_CP(bsyncr_df,
                                nmecr::assign_model_inputs(regression_type = "3PC"))
print(model$model_input_options$regression_type)
# prints "3PC"
```